spark-atlas-connector/src/main/scala/com/hortonworks/spark/atlas/types/external.scala

```diff
@@ -121,6 +121,11 @@ object external {
     }
 
     val qUri = qualifiedPath(path).toUri
+
+    // Path.makeQualified always converts the path as absolute path, but for file scheme
+    // it provides different prefix.
+    // (It provides prefix as "file" instead of "file://", which both are actually valid.)
+    // Given we have been providing it as "file://", we will keep this as it is.
```

Comment:
Maybe change it to: Given we have been providing it as "file://", the logic below changes the scheme of type "file" to "file://"

Reply:
Thanks for the suggestion. Applied.